src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -78,18 +78,23 @@ const DefaultTranscriptslist = (props: Props) => {
 
   const { gene } = props;
 
+  const filteredTranscripts = Object.values(filters).some(
+    (filter) => filter.selected
+  )
```

Comment:
Doesn't the `filterTranscripts` function take care of the condition?

Also, why do you have to specify that the return value of `filterTranscripts(gene.transcripts, filters)` is Transcript[]?

`const filteredTranscripts = filterTranscripts(gene.transcripts, filters)` should be sufficient. If there are problems with typescript, we need to look closer at the type of `filterTranscripts` (it should be a generic).

Reply:
I tried to make these changes to see if it works but there are many typescript errors now. Please let me know when you got some time and we can discuss this